source/_integrations/elkm1.markdown

```diff
@@ -553,3 +553,15 @@ Speak a word. The list of words is defined in the ElkM1 ASCII Protocol documenta
 | ---------------------- | -------- | ----------- |
 | `word` | no | Word to speak.
 | `prefix` | yes | Prefix to identify panel when multiple panels configured.
+
+## Debugging
```

Comment:
Maybe link to https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging and only include the manual config if they need it before startup happens?

Reply:
Thanks. I was aware of the UI for debug. Does it persist over restarts?

Reply:
No, it does not persist over restart.